validateForm.js

```diff
@@ -0,0 +1,19 @@
+const contactForm = document.getElementById('contactForm');
+const emailInput = contactForm.elements.email;
+function emailValidate(strng) {
+  const toLower = strng.toLowerCase();
+  if (strng === toLower) {
+    return true;
+  }
+  return false;
+}
+contactForm.addEventListener('submit', (event) => {
+  event.preventDefault();
+  if (emailValidate(emailInput.value)) {
+    contactForm.submit();
+  } else {
+    const p = document.createElement('p');
+    p.innerText = 'Please make sure the email is in lowercase.';
+    contactForm.appendChild(p);
+  }
```

Comment:
- [ ] This is a brilliant implementation :+1:  with just a few things to check to ensure that the design looks great and improves our user experience. Lets look at the :camera_flash: below and see how we can improve this :+1: 

![image](https://user-images.githubusercontent.com/99715640/216180988-7475276f-3701-4950-aafe-94330eb6c099.png)

- The error-message displays well but it is too far from the submit button and I had to check for about 2 mins before I noticed that the height has increased further down in order to display the message. I am suggesting that as per requirement, you keep it around the `email input` or very close (around) the submit button. This is good for user experience. :wink: 
- Another bug that I noticed is that the more I click the submit button, the more messages pop up and the page keeps growing in height. This is not too good a design for a solid user experience. How about using a `setTimeout()` method to delay the message for some specified seconds and then make it disappear? You can learn about it [here](https://developer.mozilla.org/en-US/docs/Web/API/setTimeout).
- To make it look even more engaging and attractive to the visitor, how about using some element of red color in the message as a sign of error? Using normal text color might make it look like part of the website at first. Don't you agree? :smile: 
- [ ] [OPTIONAL] To make your form more accessible and readable, I do think changing the text color to a more dark shade will make the user see every character more clearly to avoid mistakes. Right?